server/queues/tasks/ImportMarkdownZipTask.test.ts

```diff
@@ -0,0 +1,87 @@
+import fs from "fs";
+import path from "path";
+import { FileOperation } from "@server/models";
+import { buildFileOperation } from "@server/test/factories";
+import { flushdb } from "@server/test/support";
+import ImportMarkdownZipTask from "./ImportMarkdownZipTask";
+
+beforeEach(() => flushdb());
+
+describe("ImportMarkdownZipTask", () => {
```

Comment:
These tests are essentially copied across from what used to be in `collectionImporter.test`